domains/minhvu.json

```
@@ -0,0 +1,10 @@
+{
+    "description": "minhvu.is-a.dev : Hello, world!",
+    "owner": {
+        "username": "minhvu",
+        "email": "vdminh.box@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://minhvu.pro"
```

```suggestion
        "CNAME": "minhvu.pro"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`). Maybe you meant to use a URL record?